spring-boot-project/spring-boot-test-autoconfigure/src/main/java/org/springframework/boot/test/autoconfigure/json/JsonTestersAutoConfiguration.java

```diff
@@ -247,10 +253,15 @@ static class BasicJsonTesterRuntimeHints implements RuntimeHintsRegistrar {
 
 		@Override
 		public void registerHints(RuntimeHints hints, ClassLoader classLoader) {
+			Method initializeMethod = ReflectionUtils.findMethod(BasicJsonTester.class, "initialize", Class.class);
+
+			if (initializeMethod == null) {
+				throw new IllegalStateException("unable to find initialize method!");
+			}
+
 			ReflectionHints reflection = hints.reflection();
 			reflection.registerType(BasicJsonTester.class, MemberCategory.INVOKE_DECLARED_CONSTRUCTORS);
-			reflection.registerMethod(ReflectionUtils.findMethod(BasicJsonTester.class, "initialize", Class.class),
-					ExecutableMode.INVOKE);
+			reflection.registerMethod(initializeMethod, ExecutableMode.INVOKE);
```

Comment:
I'd rather this failed with an NPE as the initialize method should always be present.